src/Test/Bag.php

```diff
@@ -18,22 +18,22 @@ class Bag
     private $options;
 
     /**
-     * @var \PhpAb\Participation\FilterInterface
+     * @var FilterInterface
      */
     private $participationFilter;
 
     /**
-     * @var \PhpAb\Variant\ChooserInterface
+     * @var ChooserInterface
      */
     private $variantChooser;
 
     /**
-     * Bag constructor.
+     * Initializes a new instance of this class.
```

Comment:
Well, it provides value for phpDocumentor later on doesn't it? We could generate the documentation and publish it on http://phpab.github.io. If we would remove this comment and run phpDocumentor checks, phpDocumentor will give us errors because the docblock has no body.